poveste.txt

```diff
@@ -0,0 +1,13 @@
+Eram adunați într-un mănunchi de cântec mut. Fără sunet, vocile noastre se sublimau sub tensiunea care ne strângea tâmplele, străpungeau cu greu membranele individuale și se adunau undeva deasupra într-un nor de tensiune. 
+
+Strofă, refren, strofă. Cântec, reculegere, cântec. Un ritm familiar.
+
+Din tristeți izolate, să împletim duioșii care să ne lege.
+Din amintiri, văi indundate, să facem lunci fertile.
```

Comment:
"Din tristeți izolate... " începea cineva, și răspundeam cu toții în cor "...să împletim duioșii care să ne lege"

așa, sau să încercăm să scrriem o strofă ca lumea? nu mă simt inspirat pentru strofă, aș vrea să ne gândim împreună